### ISSUE TITLE
Using more than one ows:BoundingBox in coverage description breaks GetCoverage tests

### ISSUE THREAD

**Issue by bermud** [opengeospatial/ets-wcs11#4]
Seems to be another xpath issue. Having two or more boundingboxes in SpatialDomain in coverage description for a WCS 1.1.1 service makes the test concatenate the LowerCorner and UpperCorner tags.  
 
```` 
<SpatialDomain>
 <ows:BoundingBox crs=""urn:ogc:def:crs:EPSG::32610"" dimensions=""2"">
    <ows:LowerCorner>566294.686583783 4136533.35399266</ows:LowerCorner>
    <ows:UpperCorner>577472.762122489 4150496.42888048</ows:UpperCorner>
 </ows:BoundingBox>
 <ows:BoundingBox crs=""urn:ogc:def:crs:OGC::imageCRS"" dimensions=""2"">
    <ows:LowerCorner>0 0</ows:LowerCorner>
    <ows:UpperCorner>1117 1396</ows:UpperCorner>
 </ows:BoundingBox>  
    .....  
````
      
Results in the following error:  

Cannot convert string ""4150496.42888048 1117 1396"" to xs:decimal  

Happens in at least in wcs:GetCoverage_MinimalRequest and wcs:GetCoverage_MissingIdentifier. In other tests this problem is likely hidden by issue 954 (issue #2  ).

[Originally reported in the OGC CITE issue tracker: issue #956](https://portal.opengeospatial.org/services/srv_public_issues.php?call=viewIssue&issue_id=956)  

_Opened: 2014-07-15 10:34:09_

**Comment by dstenger** [comment]
We were not able to reproduce this bug. Can you please provide a service URL which leads to the described errors?

**Comment by matsoo** [comment]
I have tested on the beta site with test version 1.6 and I do not get this issue anymore. The tests still crash but I think that is because of a separate issue as the bounding box parser seems to work at least in wcs:GetCoverage_MissingIdentifier.

**Comment by dstenger** [comment]
We were able to reproduce this error with: http://192.71.92.21/MapService/wcs.axd/Dem?
and created a fix (which was tested with the URL above).
Now, the error
```
Cannot convert string ""4150496.42888048 1117 1396"" to xs:decimal
```
does not appear anymore.

A pull request for this fix was created: #14

The fix will be part of the next release.

**Comment by dstenger** [comment]
Pull request was merged successfully into master branch and is part of the upcoming release.
